src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -166,13 +166,18 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
     firstScheduleDate,
   ]);
 
+  const relatedVenueRooms =
+    relatedVenues.flatMap((venue) => venue.rooms || []).filter((x) => !!x) ||
+    [];
+  const currentVenueRooms = currentVenue?.rooms || [];
```

Comment:
`relatedVenues` should also contain the current venue, this should be unnecessary. And right now it will have duplicated rooms.

Reply:
thanks, fixed c2ecb4a